acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/TestDataSteps.java

```diff
@@ -31,6 +39,12 @@
     public void clearTestData() {
         log.debug("Disposing of test data.");
         testDataRepo.clear();
+
+        try {
+            FileUtils.deleteDirectory(acceptanceTestProperties.getDownloadDir().toFile());
```

Comment:
Won't this delete the download directory on our PCs?

Reply:
I sure hope it will! :-D

This is a directory that, by default, is located under Maven 'target' directory and is where WebDriver downloads files to during download tests. As it's being located under 'target', the content gets deleted with each `mvn clean` but in performance tests I needed those files to be gone before each test run (and I'd run those tests frequently without doing `mvn clean`) so that files downloaded during current test wouldn't have name conflicts with those left around by previous runs.

Reply:
Ah OK so it's different to our default download folder on our PC, got it 👍